react/features/filmstrip/components/web/AudioMutedIndicator.tsx

```diff
@@ -1,27 +1,25 @@
-/* @flow */
-
 import React from 'react';
 
-import { IconMicSlash } from '../../../base/icons';
-import { BaseIndicator } from '../../../base/react';
+import { IconMicSlash } from '../../../base/icons/svg';
+import BaseIndicator from '../../../base/react/components/web/BaseIndicator';
 
 /**
  * The type of the React {@code Component} props of {@link AudioMutedIndicator}.
  */
-type Props = {
+interface IProps {
 
     /**
      * From which side of the indicator the tooltip should appear from.
      */
-    tooltipPosition: string
-};
+    tooltipPosition: 'top' | 'bottom' | 'left' | 'right';
```

Comment:
nit: maybe extract in a type POSITION = 'top' | 'bottom' | 'left' | 'right' and then assign props as type `tooltipPosition : POSITION` as I see several places where this is duplicated